src/NimBLEUtils.cpp

```diff
@@ -81,6 +81,7 @@ const char* NimBLEUtils::returnCodeToString(int rc) {
     switch(rc) {
         case 0:
             return "SUCCESS";
+#ifndef NIMBLEUTILS_NO_STRINGS
```

Comment:
Could we change this to `#ifdef NIMBLE_ENABLE_DEBUG_TEXT` and move it to just before `switch(rc)`, I think it would be better if the user had to explicitly enable rather than disable the messages.

Reply:
I made it reverse, because of backward compatibility, but it's equal for me.

Reply:
Not too concerned about backward compatibility with this, just seems more natural the debugging should be enabled explicitly rather than disabled.